README.rst

```diff
@@ -73,41 +66,6 @@ Install ephypype
     cd ..
 
 
-.. _graph_install:
-
-.. 
-    Install graphpype
-    +++++++++++++++++++++++
-    
-    .. code-block:: bash 
-    
-        git clone https://github.com/neuropycon/graphpype.git
-        cd graphpype
-        pip install .
-        cd ..
-    
-    see |README_graph| for more information.
-    
-    .. |README_graph| raw:: html
-    
-       <a href="https://github.com/neuropycon/graphpype/blob/master/README.md" target="_blank">README</a>
-
-
-.. _mne_install:
-   
-Install MNE python
-++++++++++++++++++
-
-.. code-block:: bash 
-
-    git clone git://github.com/mne-tools/mne-python.git
-    cd mne-python
-    sudo python setup.py develop
-    cd ..
-
-see |MNE install| for more information.
-
-
```

Comment:
@annapasca I also removed installation instruction for other packages. I think it's not so relevant here. We should simplify things to not bring people with MNE installation problems to neuropycon